test/unit/invoice_test.rb

```diff
@@ -18,7 +18,7 @@ def buyer_vat_rate(vat_rate)
   end
 
   def setup
-    Timecop.return
+    travel_back
```

Comment:
Do we need this? Seems like a fix for some other test that didn't travel back.

Reply:
It doesn't seem we need it. I removed it,